lib/ctcs2_to_junit.pm

```diff
@@ -73,8 +73,8 @@ sub generateXML {
         failures  => "$fail_num",
         hostname  => `hostname`,
         id        => "0",
-        name      => get_var("QA_TESTSUITE"),
-        package   => get_var("QA_TESTSUITE"),
```

Comment:
I think it is better to keep variable to be more flexible for future cases

Reply:
OK